src/pages/AdvertiseData/components/TradeForm.tsx

```diff
@@ -252,6 +252,7 @@ export const TradeForm: React.FC<TradeFormProps> = (props) => {
   function validateBaseInput() {
     if (
       !(dataNFTStreamUrl.startsWith("https://") || dataNFTStreamUrl.startsWith("ipns://")) ||
+      dataNFTStreamUrl.startsWith("ipfs://") ||
```

Comment:
This seems wrong. This should be:
!(dataNFTStreamUrl.startsWith("https://") || dataNFTStreamUrl.startsWith("ipns://") || dataNFTStreamUrl.startsWith("ipfs://"))